ge.xml

```diff
@@ -2545,7 +2542,6 @@
           <key key="contact:facebook" value="https://www.facebook.com/IsbankGeorgia/" />
           <key key="contact:email" value="info@isbank.ge" />
           <key key="name:ka" value="იშბანკი" />
-          <key key="contact:email" value="info@rebank.ge" />
```

Comment:
This is the line that is most likely to be wrong. L2546 actually has the bank name in it though.